.eslintrc.json

```diff
@@ -26,6 +26,7 @@
         "rules": {
           "one-var": 0,
           "one-var-declaration-per-line": 0,
+          "arrow-body-style": [0, "always"],
```

Comment:
@kleva-j why are we ignoring this?

Reply:
@kleva-j when I ask these questions. I expect an answer, so I can know the reason behind the action.

Reply:
It was an issue with the previous implementation but I have resolved it. Different Eslint configurations were affecting each other. When I resolve one it affects the other.
I have fixed it.